sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -0,0 +1,928 @@
+shown in a single column in the table below.
+
+[frame="none",options="header"]
+|======================
+| A type | B type | C and D type | M | N | K | device
+| `matrix_type::uint8`  | `matrix_type::uint8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+| `matrix_type::uint8`  | `matrix_type::int8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+| `matrix_type::int8`  | `matrix_type::uint8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+| `matrix_type::int8`  | `matrix_type::int8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+|  `matrix_type::fp16`       |  `matrix_type::fp16`   |
+`matrix_type::fp32`   |  +<=+ 8 |  16   |  16 | architecture::intel_gpu_pvc
+| | | | |8|| architecture::intel_gpu_dg2
+|  `matrix_type::bf16`       |  `matrix_type::bf16`   |
+`matrix_type::fp32`   |  +<=+ 8 |  16   |  16 | architecture::intel_gpu_pvc
+| | | | |8|| architecture::intel_gpu_dg2
+|======================
+
+==== Nvidia Tensor Cores Supported Combinations
+
+The complete set of matrix data types and shapes that are supported by
+the `ext_oneapi_cuda` backend are represented in the following
+table. Tm indicates the matrix element data type held by a
+"multiplicand" `joint_matrix`: i.e requiring `use::a` or `use::b`. Tc
+indicates the matrix element data type held by an "accumulator"
+`joint_matrix`: i.e requiring `use::accumulator`.
+
+IMPORTANT: When compiling for the `ext_oneapi_cuda` backend the target
+arch backend flag, `-Xsycl-target-backend --cuda-gpu-arch=sm_xx`, must
+be used, where `sm_xx` must be a Compute Capability that is equal to
+or greater than the appropriate Minimum Compute Capability. When an
+executable has been compiled for `sm_xx`, if the executable is run on
+a device with compute capability less than `sm_xx` then an error will
+be thrown. The mapping to Minimum Compute Capability from each
+supported parameter combination is specified in the following table.
+
+--
+[.center]
+|======================
+|Tm (`use::a` or `use::b`) |Tc (`use::accumulator`) |M |N |K | Minimum Compute Capability
+.3+|half  .3+|float
+|16 |16 |16 .6+| sm_70
+|8 |32 |16
+|32 |8 |16
+.3+|half  .3+|half
+|16 |16 |16
+|8 |32 |16
+|32 |8 |16
+.3+|int8_t  .3+|int32_t
+|16 |16 |16 .6+| sm_72
+|8 |32 |16
+|32 |8 |16
+.3+|uint8_t  .3+|int32_t
+|16 |16 |16
+|8 |32 |16
+|32 |8 |16
+|precision::tf32  |float |16 |16 |8 .5+| sm_80
+.3+|bfloat16  .3+|float
+|16 |16 |16
+|8 |32 |16
+|32 |8 |16
+|double  |double |8 |8 |4
+|======================
+--
+
+The M, N, K triple from the above table defines the complete set of
+matrix shapes constructible:
+--
+[.center]
+|======================
+|use |NumRows | NumCols
+|a |M |K
+|b |K |N
+|accumulator | M| N
+|======================
+--
```

Comment:
In that case, let's remove the table and the preceding paragraph "The M, N, K triple from the above table defines the complete set of matrix shapes constructible:".  They don't provide any useful information since the terms "M", "N", and "K" are already defined at the start of the section.